profile/README.md

```diff
@@ -14,3 +14,149 @@ The OSTIS technology is implemented as a network of ostis-systems, which is part
+
+- Due to the fact that the entire system is described by means of the SC-code, it can analyze itself, look for errors in itself and optimize its own work, i.e. has reflexivity. Reflexivity is considered one of the key signs of **intelligence**.
+
+- Due to the presence of the basic alphabet of the SC-code and the possibility of a complete description of the system by means of the SC-code, it becomes possible to make ostis-systems completely platform-independent. That is, the development of an ostis-system comes down to the development of its model. The platform, in turn, can be implemented both in software (like a virtual machine) and in hardware.
+
+- OSTIS is the basis for a new type of computers – semantic computers. Uncommon computers with unconventional architecture (including supercomputers) are often difficult to use due to the lack of software. The OSTIS Technology and ostis-systems will run on semantic computers, easing the migration to the next generation of computers.
+
+- Due to the special kind of multi-agent information processing used in OSTIS, ostis-systems are initially focused on parallel processing of information (including its support at the hardware level within a semantic computer).
+
+## Key takeaways
+
+OSTIS is not a specific intelligent system or a method for solving problems of any class; it is a **technology** for developing intelligent systems, each of which, in turn, will solve problems of a certain class.
+
+The key advantages of OSTIS lie not in fundamentally new functional capabilities of the systems being developed (most ostis-systems features can be implemented with traditional tools), but in how easy it is to modify and develop the systems, adapt them to new tasks as well as how efficiently it is possible to accumulate and use the obtained components in the development of new systems, while reducing the time and labor intensity of the development process.
+
+OSTIS is a way to solve the compatibility problem, one of the most important problems of modern technologies.
+
+## List of repositories developed by the project
+
+- [OSTIS Technology Standard](https://github.com/ostis-ai/ostis-standard) - OSTIS Technology documentation represented in the SCn-code
+- [OSTIS Web-oriented software platform](https://github.com/ostis-ai/ostis-web-platform)
+  - [Software implementation of sc-memory](https://github.com/ostis-ai/sc-machine)
+
+  - [Source code of Intelligent Metasystem knowledge base](https://github.com/ostis-ai/ims.ostis.kb)
+
+  - [Web oriented sc-models interpreter](https://github.com/ostis-ai/sc-web)
+
+  - [Official API client (Python)](https://github.com/ostis-ai/py-sc-client)
+
+  - [Official API client (TypeScript)](https://github.com/ostis-ai/ts-sc-client)
+
```

Comment:
add sc-component-manager and scl-machine!!!!!!!

Reply:
Not fixed

Reply:
fixed